client/am/console/src/main/resources/org/apache/syncope/client/console/clientapps/ClientAppDirectoryPanel.properties

```diff
@@ -65,3 +65,4 @@ privacyUrl=Privacy URL
 compose.title=username attribute provider
 usernameAttributeProviderConf.title=Username Attribute Provider for ${name}
 ticketExpirationPolicy=Ticket Expiration Policy
+menu.history=Configuiration History
```

Comment:
Fix typo (all languages)

Reply:
Also, copy the translation from other properties, to keep consistency, for example `TopologyTogglePanel*.properties`